lemarche/www/pages/views.py

```diff
@@ -183,6 +188,39 @@ def limit_list(self, listing: list, limit: int = 3, with_end_elmt=True, end_posi
         return listing
 
 
+def calculate_social_impact_buyers(amount: int):
+    AMOUNT_MIN = 3699
```

Comment:
```suggestion
    AMOUNT_THRESHOLD = 3699
```
au début j'ai lu `MIN` en pensant qu'en dessous il ne se passerait rien ^^

Reply:
bien vu